cmake/MacOSXBundleInfo.plist.in

```diff
@@ -36,5 +36,7 @@
     <true/>
     <key>NSHumanReadableCopyright</key>
     <string>${MACOSX_BUNDLE_COPYRIGHT}</string>
+    <key>NSMicrophoneUsageDescription</key>
+    <string>MultiMC does not need access to your microphone for itself. Likely it's requesting it on behalf of a Minecraft mod you installed, which needs to access your microphone in order to function properly.</string>
```

Comment:
I tried to hit all the main points in this explanation message, but I think it could be better worded. Please suggest improvements.

Reply:
Maybe something like this is better?:

> Your microphone is not needed by MultiMC itself. However, you might have installed a mod that needs access to your microphone in order to function properly, and so MultiMC is requesting for the permission on behalf of the mod. If it's not the case, and if you're not sure why your microphone is needed, then it's safer if you decline this request.

Reply:
I think the current wording is pretty good. Let's go with it.